examples/sveltekit/__integration-tests__/sveltekit.spec.ts

```diff
@@ -26,7 +26,9 @@ describe('SvelteKit integration', () => {
 			// Kill the port if it's used!
 			try {
 				execSync('fuser -k 3007/tcp');
-			} catch (error) {}
+			} catch (error) {
```

Comment:
Sveltekit tests shouldn't work in Node 14.

Reply:
yeah, see jest.config.js, they are only enabled conditionally

Reply:
I think they still run on Node 14;
See https://github.com/dotansimha/graphql-yoga/runs/7765563141?check_suite_focus=true#step:7:33

Reply:
fixed in https://github.com/dotansimha/graphql-yoga/pull/1567/commits/86cdea7186edfbcadaca4326c793ca4ba1c24bec